pyinstrument/middleware.py

```diff
@@ -32,6 +33,10 @@ def process_response(self, request, response):
             # high and the other parts (see line 36) are to be taken into account; so a hundred will be fine here).
             path = request.get_full_path().replace('/', '_')[:100]
 
+            # Swap ? for _qs_ on Windows, as it does not support ? in filenames.
+            if platform.platform().startswith('Windows'):
```

Comment:
Sorry to be difficult (I can see you've been around the houses on this!) but the docs say about `platform.platform()` "The output is intended to be human readable rather than machine parseable"

I think `sys.platform` might be the best approach here. How about `if sys.platform in ['win32', 'cygwin']:`?

Reply:
Seems totally reasonable to me :) Coming soon!

Reply:
Updated - thanks for sticking with it!